### PR TITLE
docs(schema): improve Cube View documentation

### PR DESCRIPTION
**Pull request by hassankhan**

### CODE REVIEW

File: docs/content/Schema/Reference/view.mdx [hunk 1]
```diff
@@ -7,25 +7,31 @@ menuOrder: 8.1
...
 
-Views **can't** have their own members. You can use `includes` property to include measures and dimensions into view from cubes.
+[![](https://mermaid.ink/img/pako:eNp1kt9LwzAQx_-VIzL2skKH-LAIgnWKbyKbvrR7yJqrDWubkh9uY-x_N0m13XQ2FO4-9819014OJJccCSWj0QFEIwyFA4yLSm7zkikzDmlu1Se6cLxmWugxHP0ajbImaz4Ua0tYzl1sXi2qvQ_8q-26q70L3GpPwD0-eaxbs08DXp3y-3RRirZFDi-KozovJumz-HDNWGURHqw2sv6RYMOzxut-GT_YNfbGPumMA16d8vv0l6GHSfqm_3EI3wlRFN11577AEs9CsWPB5gJLur5hywVlcqbs3POKaT3HApbCVAiFqCp6NbueTYt8Fk9gLXeRLhmXWwo37Q6mcbubQC4rqSh86-L4tu_U_aau2QkN4xloOGXv_MQ2-Nc9du7aKLnBaCu4KSl4m37jMIJh_2m5vxpDmUyIm3LNBHcX9ODFGTEl1pgR6kLO1CYjWXN0OmaNXOybnFCjLE6IbTkzOBfM3YWa0IJVGo9fIZX7Vg)](https://mermaid.live/edit#pako:eNp1kt9LwzAQx_-VIzL2skKH-LAIgnWKbyKbvrR7yJqrDWubkh9uY-x_N0m13XQ2FO4-9819014OJJccCSWj0QFEIwyFA4yLSm7zkikzDmlu1Se6cLxmWugxHP0ajbImaz4Ua0tYzl1sXi2qvQ_8q-26q70L3GpPwD0-eaxbs08DXp3y-3RRirZFDi-KozovJumz-HDNWGURHqw2sv6RYMOzxut-GT_YNfbGPumMA16d8vv0l6GHSfqm_3EI3wlRFN11577AEs9CsWPB5gJLur5hywVlcqbs3POKaT3HApbCVAiFqCp6NbueTYt8Fk9gLXeRLhmXWwo37Q6mcbubQC4rqSh86-L4tu_U_aau2QkN4xloOGXv_MQ2-Nc9du7aKLnBaCu4KSl4m37jMIJh_2m5vxpDmUyIm3LNBHcX9ODFGTEl1pgR6kLO1CYjWXN0OmaNXOybnFCjLE6IbTkzOBfM3YWa0IJVGo9fIZX7Vg)
```
Comment (hassankhan) [inline docs/content/Schema/Reference/view.mdx:14]: Will be replacing this image with a nicer one

File: docs/content/Schema/Fundamentals/Working-with-Joins.mdx [hunk 2]
```diff
@@ -517,6 +531,72 @@ cube('Customers', {
...
+      type: `number`,
+    },
+  },
+
+  dimensions: {
+    customer: {
+      sql: `${Users.company}`,
+      type: `string`,
+    },
+  },
```
Comment (hassankhan) [inline docs/content/Schema/Fundamentals/Working-with-Joins.mdx:576]: These can probably be `includes` instead, but there seems to be a bug in Cube preventing me from testing this right now

File: docs/content/Schema/Reference/view.mdx [hunk 3]
```diff
@@ -7,25 +7,38 @@ menuOrder: 8.1
 subCategory: Reference
 ---
 
-Views sit on top of the data graph of cubes and create a facade of your whole data model with which data consumers can interact. 
-They’re the place to define metrics, manage governance and data access, and control ambiguous join paths.
-
-Views **can't** have their own members. You can use `includes` property to include measures and dimensions into view from cubes.
+Views sit on top of the data graph of cubes and create a facade of your whole
+data model with which data consumers can interact. They provide single place to
```
Comment (rpaik) [inline docs/content/Schema/Reference/view.mdx:11]:
```suggestion
data model that data consumers can interact with. They provide a single place to
```

File: docs/content/Schema/Reference/view.mdx [hunk 4]
```diff
@@ -67,13 +81,16 @@ FROM CompletedOrders
...
+pre-aggregations defined in the underlying cubes used by the view.
+
 ## Parameters
 
 ### <--{"id" : "Parameters"}--> description
 
-Use a description in your cubes to allow your team to better understand what
-this view is about. It is a very simple and yet useful tool that gives a hint to
-everyone and makes sure data is interpreted correctly by users.
+A description of the view to allow your team to better understand what its
```
Comment (rpaik) [inline docs/content/Schema/Reference/view.mdx:91]:
```suggestion
A description of the view allows your team to better understand what its
```

File: docs/content/Schema/Reference/view.mdx [hunk 5]
```diff
@@ -67,13 +81,16 @@ FROM CompletedOrders
...
+purpose is. It is a very simple and yet useful tool that gives a hint to
+everyone and ensures data is interpreted correctly by users.
```
Comment (rpaik) [inline docs/content/Schema/Reference/view.mdx:93]:
```suggestion
everyone and ensures that data is interpreted correctly by users.
```

**Comment by rpaik** [comment]
A few nits 😄

**Comment by keydunov** [comment]
Tagging @paveltiunov for review

**Pull request merged**